org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/DiagnosticResolution.java

```diff
@@ -65,11 +75,17 @@ public DiagnosticResolution(String label, Factory modificationContextFactory, IT
 		this.relevance = relevance;
 	}
 
-	public DiagnosticResolution(String label, Factory modificationContextFactory, IModification<EObject> modification) {
+	/**
+	 * @since 2.27
+	 */
+	public DiagnosticResolution(String label, Factory modificationContextFactory, ISemanticModification modification) {
 		this(label, modificationContextFactory, modification, 0);
 	}
 
-	public DiagnosticResolution(String label, Factory modificationContext, IModification<EObject> modification,
+	/**
+	 * @since 2.27
+	 */
+	public DiagnosticResolution(String label, Factory modificationContext, ISemanticModification modification,
```

Comment:
Curiosity: What is the meaning of the `relevance`? Mind adding som docs to the field and getter since you're here?

Reply:
I also do not know for sure. @tivervac suggested I should add it to the API in https://github.com/eclipse/xtext-core/pull/1881#discussion_r852956806 and I did it (and continue now), as previous methods had such a parameter.
I could guess that it could be used to sort the list of QFs which are proposed, but as far as I see, the Xtext LS does not do anything useful with it. @tivervac , do you know what is it for?

Reply:
I do not. Sounds like some kind of priority metric to me

Reply:
looks like a copy from org.eclipse.xtext.ui.editor.quickfix.IssueResolution.relevance